model/span.go

```diff
@@ -110,6 +111,28 @@ func (s *Span) IsRPCServer() bool {
 	return s.HasSpanKind(SpanKindServer)
 }
 
+// GetHashTag returns hash keyValue from Tags and wether the hash was found
+func (s *Span) GetHashTag() (hashCode int64, found bool) {
+	if tag, ok := KeyValues(s.Tags).FindByKey(SpanHashKey); ok {
+		return tag.GetVInt64(), true
+	}
+	return -1, false
```

Comment:
```suggestion
	return 0, false
```
I don't thin `-1` conveys any more meaning than 0, so I would rather return default value, this would match behavior of a get from standard `map[string]int64`